src/client/Game.jsx

```diff
@@ -78,12 +78,12 @@ export default class Game extends React.Component {
     this.ws = new window.WebSocket(`ws://${window.location.host}/api?token=${token}`)
     this.ws.onmessage = (event) => {
       const message = JSON.parse(event.data)
```

Comment:
We can not know for sure what event contains here until we know which type of message it is.

Reply:
Refactored out each part of event.data in separate variables based on switch case as it's a common scope.